packages/compass-e2e-tests/helpers/commands/get-connect-form-state.ts

```diff
@@ -340,13 +340,8 @@ async function getMultipleValues(
   browser: CompassBrowser,
   selector: string
 ): Promise<string[] | null> {
-  const elements = await browser.$$(selector);
   const results = (
-    await Promise.all(
-      elements.map((element) => {
-        return element.getValue();
-      })
-    )
+    await browser.$$(selector).map((element) => element.getValue())
```

Comment:
There are quite a few of these because we use the same pattern in a bunch of places. Basically the Element and ElementArray (I think that's what it is called) types have changed in the new webdriver. Notably they have their own map that already does the whole Promise.all() step for you. Which is convenient but was super confusing to debug :)